examples/pylint/w0644_unbalanced_dict_unpacking.py

```diff
@@ -0,0 +1,12 @@
+"""Examples for W0644 unbalanced-dict-unpacking"""
+
+SCORES = {
+    "bob": (1, 1, 3, 2),
+    "joe": (4, 3, 1, 2),
+    "billy": (2, 2, 2, 2),
+}
+
+a, b = SCORES   # unbalanced-dict-unpacking
```

Comment:
Similar to my review on your other PR, this is good but the comment should just be `Error on this line`. Also please use exactly 2 spaces before the `#`